### ISSUE TITLE
Build fails on i686

### ISSUE THREAD

**Issue by hsivonen** [rust-lang-nursery/packed_simd#206]
# Steps to reproduce

Try to build a crate that depends on `packed_simd` (either the git tip or the crates.io release 0.3.1) on nightly-i686-unknown-linux-gnu rustc 1.33.0-nightly (4c2be9c97 2019-01-22).

# Actual results

```
error[E0432]: unresolved imports `crate::codegen::pointer_sized_int::isize_`, `crate::codegen::pointer_sized_int::usize_`
 --> /home/hsivonen/Projects/packed_simd/src/codegen/vSize.rs:3:41
  |
3 | use crate::codegen::pointer_sized_int::{isize_, usize_};
  |                                         ^^^^^^  ^^^^^^ no `usize_` in `codegen::pointer_sized_int`
  |                                         |
  |                                         no `isize_` in `codegen::pointer_sized_int`

error[E0433]: failed to resolve: did you mean `core::arch`?
  --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:50:20
   |
50 |           use crate::arch::$arch::{
   |                      ^^^^ did you mean `core::arch`?
...
86 | / impl_arch!(
87 | |     [x86["x86"]: __m64], [x86_64["x86_64"]: __m64],
88 | |     [arm["arm"]: int8x8_t, uint8x8_t, poly8x8_t, int16x4_t, uint16x4_t,
89 | |      poly16x4_t, int32x2_t, uint32x2_t, float32x2_t, int64x1_t,
...  |
96 | |     test: test_v64
97 | | );
   | |__- in this macro invocation

error[E0433]: failed to resolve: did you mean `core::arch`?
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:50:20
    |
50  |           use crate::arch::$arch::{
    |                      ^^^^ did you mean `core::arch`?
...
110 | / impl_arch!(
111 | |     [x86["x86"]: __m128, __m128i, __m128d],
112 | |     [x86_64["x86_64"]:  __m128, __m128i, __m128d],
113 | |     [arm["arm"]: int8x16_t, uint8x16_t, poly8x16_t, int16x8_t, uint16x8_t,
...   |
129 | |     test: test_v128
130 | | );
    | |__- in this macro invocation

error[E0433]: failed to resolve: did you mean `core::arch`?
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:50:20
    |
50  |           use crate::arch::$arch::{
    |                      ^^^^ did you mean `core::arch`?
...
178 | / impl_arch!(
179 | |     [x86["x86"]: __m256, __m256i, __m256d],
180 | |     [x86_64["x86_64"]:  __m256, __m256i, __m256d] |
181 | |     from: i8x32, u8x32, m8x32, i16x16, u16x16, m16x16,
...   |
186 | |     test: test_v256
187 | | );
    | |__- in this macro invocation

error[E0432]: unresolved imports `crate::codegen::pointer_sized_int::isize_`, `crate::codegen::pointer_sized_int::usize_`
 --> /home/hsivonen/Projects/packed_simd/src/vSize.rs:3:41
  |
3 | use crate::codegen::pointer_sized_int::{isize_, usize_};
  |                                         ^^^^^^  ^^^^^^ no `usize_` in `codegen::pointer_sized_int`
  |                                         |
  |                                         no `isize_` in `codegen::pointer_sized_int`

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/lib.rs:240:1
    |
240 | / cfg_if! {
241 | |     if #[cfg(all(target_arch = "arm", target_feature = "v7", target_feature = "neon",
242 | |                  feature = "coresimd"))] {
243 | |         #[allow(unused_imports)]
...   |
248 | |     }
249 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/abs.rs:41:1
    |
41  | / cfg_if! {
42  | |     if #[cfg(target_arch = "s390x")] {
43  | |         // FIXME: https://github.com/rust-lang-nursery/packed_simd/issues/14
44  | |         impl_unary!(f32x2[f32; 2]: fabs_f32);
...   |
102 | |     }
103 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/cos.rs:41:1
    |
41  | / cfg_if! {
42  | |     if #[cfg(target_arch = "s390x")] {
43  | |         // FIXME: https://github.com/rust-lang-nursery/packed_simd/issues/14
44  | |         impl_unary!(f32x2[f32; 2]: cos_f32);
...   |
102 | |     }
103 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/cos_pi.rs:35:1
   |
35 | / cfg_if! {
36 | |     if #[cfg(all(target_arch = "x86_64", feature = "sleef-sys"))] {
37 | |         use sleef_sys::*;
38 | |         cfg_if! {
...  |
86 | |     }
87 | | }
   | |_^
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/exp.rs:41:1
    |
41  | / cfg_if! {
42  | |     if #[cfg(target_arch = "s390x")] {
43  | |         // FIXME: https://github.com/rust-lang-nursery/packed_simd/issues/14
44  | |         impl_unary!(f32x2[f32; 2]: exp_f32);
...   |
111 | |     }
112 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/ln.rs:41:1
    |
41  | / cfg_if! {
42  | |     if #[cfg(target_arch = "s390x")] {
43  | |         // FIXME: https://github.com/rust-lang-nursery/packed_simd/issues/14
44  | |         impl_unary!(f32x2[f32; 2]: ln_f32);
...   |
111 | |     }
112 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/mul_add.rs:36:1
    |
36  | / cfg_if! {
37  | |     if #[cfg(target_arch = "s390x")] {
38  | |         // FIXME: https://github.com/rust-lang-nursery/packed_simd/issues/14
39  | |         macro_rules! impl_broken {
...   |
108 | |     }
109 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/powf.rs:41:1
    |
41  | / cfg_if! {
42  | |     if #[cfg(target_arch = "s390x")] {
43  | |         // FIXME: https://github.com/rust-lang-nursery/packed_simd/issues/14
44  | |         impl_binary!(f32x2[f32; 2]: powf_f32);
...   |
111 | |     }
112 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/sin.rs:41:1
    |
41  | / cfg_if! {
42  | |     if #[cfg(target_arch = "s390x")] {
43  | |         // FIXME: https://github.com/rust-lang-nursery/packed_simd/issues/14
44  | |         impl_unary!(f32x2[f32; 2]: sin_f32);
...   |
102 | |     }
103 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/sin_cos_pi.rs:143:1
    |
143 | / cfg_if! {
144 | |     if #[cfg(all(target_arch = "x86_64", feature = "sleef-sys"))] {
145 | |         use sleef_sys::*;
146 | |         cfg_if! {
...   |
194 | |     }
195 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/sin_pi.rs:35:1
   |
35 | / cfg_if! {
36 | |     if #[cfg(all(target_arch = "x86_64", feature = "sleef-sys"))] {
37 | |         use sleef_sys::*;
38 | |         cfg_if! {
...  |
86 | |     }
87 | | }
   | |_^
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/sqrt.rs:41:1
    |
41  | / cfg_if! {
42  | |     if #[cfg(target_arch = "s390x")] {
43  | |         // FIXME: https://github.com/rust-lang-nursery/packed_simd/issues/14
44  | |         impl_unary!(f32x2[f32; 2]: sqrt_f32);
...   |
102 | |     }
103 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/math/float/sqrte.rs:15:1
   |
15 | / cfg_if! {
16 | |     if #[cfg(all(target_arch = "x86_64", feature = "sleef-sys"))] {
17 | |         use sleef_sys::*;
18 | |         cfg_if! {
...  |
66 | |     }
67 | | }
   | |_^
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:21:1
   |
21 | / cfg_if! {
22 | |     if #[cfg(any(target_arch = "x86", target_arch = "x86_64"))] {
23 | |         #[macro_use]
24 | |         mod x86;
...  |
35 | |     }
36 | | }
   | |_^
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:38:1
   |
38 | impl_mask_reductions!(m8x2);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:39:1
   |
39 | impl_mask_reductions!(m8x4);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:40:1
   |
40 | impl_mask_reductions!(m8x8);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:41:1
   |
41 | impl_mask_reductions!(m8x16);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:42:1
   |
42 | impl_mask_reductions!(m8x32);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:43:1
   |
43 | impl_mask_reductions!(m8x64);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:45:1
   |
45 | impl_mask_reductions!(m16x2);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:46:1
   |
46 | impl_mask_reductions!(m16x4);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:47:1
   |
47 | impl_mask_reductions!(m16x8);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:48:1
   |
48 | impl_mask_reductions!(m16x16);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:49:1
   |
49 | impl_mask_reductions!(m16x32);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:51:1
   |
51 | impl_mask_reductions!(m32x2);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:52:1
   |
52 | impl_mask_reductions!(m32x4);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:53:1
   |
53 | impl_mask_reductions!(m32x8);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:54:1
   |
54 | impl_mask_reductions!(m32x16);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:58:1
   |
58 | impl_mask_reductions!(m64x2);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:59:1
   |
59 | impl_mask_reductions!(m64x4);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:60:1
   |
60 | impl_mask_reductions!(m64x8);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:62:1
   |
62 | impl_mask_reductions!(m128x1);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:63:1
   |
63 | impl_mask_reductions!(m128x2);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:64:1
   |
64 | impl_mask_reductions!(m128x4);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:66:1
   |
66 | impl_mask_reductions!(msizex2);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:67:1
   |
67 | impl_mask_reductions!(msizex4);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `impl_mask_reductions!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/reductions/mask.rs:68:1
   |
68 | impl_mask_reductions!(msizex8);
   | ^^^^^^^^^^^^^^^^^^^^

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/shuffle1_dyn.rs:293:1
    |
293 | impl_shuffle1_dyn!(u8x8);
    | ^^^^^^^^^^^^^^^^^^^^^^^^^ in this macro invocation
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/shuffle1_dyn.rs:294:1
    |
294 | impl_shuffle1_dyn!(u8x16);
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^ in this macro invocation
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/shuffle1_dyn.rs:305:1
    |
305 | impl_shuffle1_dyn!(u32x4);
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^ in this macro invocation
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/shuffle1_dyn.rs:309:1
    |
309 | impl_shuffle1_dyn!(u64x2);
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^ in this macro invocation
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/swap_bytes.rs:169:1
    |
169 | / cfg_if! {
170 | |     if #[cfg(target_pointer_width = "8")] {
171 | |         impl_swap_bytes!(v16: isizex2, usizex2,);
172 | |         impl_swap_bytes!(v32: isizex4, usizex4,);
...   |
188 | |     }
189 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/codegen/bit_manip.rs:322:1
    |
322 | / cfg_if! {
323 | |     if #[cfg(target_pointer_width = "8")] {
324 | |         impl_bit_manip! { sized: usizex2, usize, isizex2, isize, u8x2 }
325 | |         impl_bit_manip! { sized: usizex4, usize, isizex4, isize, u8x4 }
...   |
341 | |     }
342 | | }
    | |_^
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: cannot find macro `__cfg_if_items!` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/codegen/pointer_sized_int.rs:5:1
   |
5  | / cfg_if! {
6  | |     if #[cfg(target_pointer_width = "8")] {
7  | |         crate type isize_ = i8;
8  | |         crate type usize_ = u8;
...  |
27 | |     }
28 | | }
   | |_^
   |
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error[E0412]: cannot find type `__m64` in this scope
  --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:87:18
   |
87 |     [x86["x86"]: __m64], [x86_64["x86_64"]: __m64],
   |                  ^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
   |
8  | use core::arch::x86::__m64;
   |

error[E0412]: cannot find type `__m128` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:111:18
    |
111 |     [x86["x86"]: __m128, __m128i, __m128d],
    |                  ^^^^^^
help: a struct with a similar name exists
    |
111 |     [x86["x86"]: m128, __m128i, __m128d],
    |                  ^^^^
help: possible candidate is found in another module, you can import it into scope
    |
8   | use core::arch::x86::__m128;
    |

error[E0412]: cannot find type `__m128i` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:111:26
    |
111 |     [x86["x86"]: __m128, __m128i, __m128d],
    |                          ^^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
    |
8   | use core::arch::x86::__m128i;
    |

error[E0412]: cannot find type `__m128d` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:111:35
    |
111 |     [x86["x86"]: __m128, __m128i, __m128d],
    |                                   ^^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
    |
8   | use core::arch::x86::__m128d;
    |

error[E0412]: cannot find type `__m256` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:179:18
    |
179 |     [x86["x86"]: __m256, __m256i, __m256d],
    |                  ^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
    |
8   | use core::arch::x86::__m256;
    |

error[E0412]: cannot find type `__m256i` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:179:26
    |
179 |     [x86["x86"]: __m256, __m256i, __m256d],
    |                          ^^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
    |
8   | use core::arch::x86::__m256i;
    |

error[E0412]: cannot find type `__m256d` in this scope
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:179:35
    |
179 |     [x86["x86"]: __m256, __m256i, __m256d],
    |                                   ^^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
    |
8   | use core::arch::x86::__m256d;
    |

error: unused import: `$arch_ty`
  --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:51:15
   |
51 |               $($arch_ty),*
   |                 ^^^^^^^^
...
86 | / impl_arch!(
87 | |     [x86["x86"]: __m64], [x86_64["x86_64"]: __m64],
88 | |     [arm["arm"]: int8x8_t, uint8x8_t, poly8x8_t, int16x4_t, uint16x4_t,
89 | |      poly16x4_t, int32x2_t, uint32x2_t, float32x2_t, int64x1_t,
...  |
96 | |     test: test_v64
97 | | );
   | |__- in this macro invocation
   |
note: lint level defined here
  --> /home/hsivonen/Projects/packed_simd/src/lib.rs:235:9
   |
235| #![deny(warnings, rust_2018_idioms)]
   |         ^^^^^^^^
   = note: #[deny(unused_imports)] implied by #[deny(warnings)]

error: unused imports: `$arch_ty`, `$arch_ty`, `$arch_ty`
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:51:15
    |
51  |               $($arch_ty),*
    |                 ^^^^^^^^
...
110 |   impl_arch!(
    |  _-
    | |_|
    | |_|
    | |
111 | |     [x86["x86"]: __m128, __m128i, __m128d],
112 | |     [x86_64["x86_64"]:  __m128, __m128i, __m128d],
113 | |     [arm["arm"]: int8x16_t, uint8x16_t, poly8x16_t, int16x8_t, uint16x8_t,
...   |
129 | |     test: test_v128
130 | | );
    | |  -
    | |__|
    | |__in this macro invocation
    | |__in this macro invocation
    |    in this macro invocation

error: unused imports: `$arch_ty`, `$arch_ty`, `$arch_ty`
   --> /home/hsivonen/Projects/packed_simd/src/api/into_bits/arch_specific.rs:51:15
    |
51  |               $($arch_ty),*
    |                 ^^^^^^^^
...
178 |   impl_arch!(
    |  _-
    | |_|
    | |_|
    | |
179 | |     [x86["x86"]: __m256, __m256i, __m256d],
180 | |     [x86_64["x86_64"]:  __m256, __m256i, __m256d] |
181 | |     from: i8x32, u8x32, m8x32, i16x16, u16x16, m16x16,
...   |
186 | |     test: test_v256
187 | | );
    | |  -
    | |__|
    | |__in this macro invocation
    | |__in this macro invocation
    |    in this macro invocation
```

# Expected results

Expected the crate to build.

**Comment by hsivonen** [comment]
I don't see this when building for the same target on an x86_64 host.

**Comment by gnzlbg** [comment]
I've released version 0.3.2 of `packed_simd`. Everything should work now. Please re-open this issue if that doesn't solve the problem for you.
